eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -48,17 +88,66 @@ extends:
           TestTimeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
           TestProxy: ${{ parameters.TestProxy }}
           MatrixConfigs:
-          - ${{ each config in parameters.MatrixConfigs }}:
-            - ${{ config }}
-          - ${{ each config in parameters.AdditionalMatrixConfigs }}:
-            - ${{ config }}
+          - ${{ parameters.MatrixConfigs }}
+          - ${{ parameters.AdditionalMatrixConfigs }}
           MatrixFilters: ${{ parameters.MatrixFilters }}
           MatrixReplace: ${{ parameters.MatrixReplace }}
 
-    - template: archetype-rust-release.yml
-      parameters:
-        DependsOn: "Build"
-        ServiceDirectory: ${{ parameters.ServiceDirectory }}
-        Artifacts: ${{ parameters.Artifacts }}
-        TestPipeline: ${{ eq(parameters.ServiceDirectory, 'template') }}
-        PipelineArtifactName: packages
+    # Run live tests for internal only, not public CI builds. This can be triggered manually for via an `/azp run` comment.
+    - ${{if eq(parameters.RunLiveTests, 'true') }}:
```

Comment:
Are you sure that's true? IIRC, `/azp run {pipeline}` still treats it as a PR trigger.

Reply:
It does. It that an issue?

Reply:
As we talked offline, that shouldn't happen in the pullrequest.yml, so cool.